demo/budg/dashboard/src/components/FilesList.js

```diff
@@ -1,164 +1,55 @@
-import React, { Component } from 'react';
```

Comment:
The `FilesList.js` could maybe also go in `pages/files`. It's not quite clear otherwise why some components are left in components and others are in pages.

Reply:
Some are `Route` components, the others are not, that's the only distinction right now.